cypress/integration/details_spec.js

```diff
@@ -26,13 +18,20 @@ describe('Money Plane Details view', () => {
       .then((mockMoneyPlaneData) => {
         cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/694919/videos', {
           statusCode: 200,
+          delay: 100,
           body: mockMoneyPlaneData
         })
       })
 
     cy.visit(baseUrl)
   });
 
+  it ('Should show a loading status while fetching movie details', () => {
+    cy
+      .get('.loading').should('contain', 'Loading')
+      .should('be.visible')
+  });
+
```

Comment:
Helpful - thanks for adding this and showing me how to incorporate that delay (pretty straight-forward!)